ledger/src/blockstore.rs

```diff
@@ -1892,7 +1892,14 @@ impl Blockstore {
                     .unwrap()
                     .map(Cow::Owned)
             })
-            .expect("merkle root meta must exist for erasure meta");
+        else {
+            warn!(
+                "The merkle root meta for the previous erasure set {prev_erasure_set:?} does not exist.
+                This should only happen if you have recently upgraded from a version < v1.18.12.
```

Comment:
```suggestion
                This should only happen if you have recently upgraded from a version < v1.18.13.
```

too late to ship this in .12